tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -0,0 +1,120 @@
+*** Settings ***
+Resource            ../libs/public-common.robot
+
+Suite Setup         user opens the browser
+Suite Teardown      user closes the browser
+
+Force Tags          GeneralPublic    Prod
+
+*** Test Cases ***
+Go to 'data-tables' page
+    user navigates to data tables page on public frontend
+
+Check page contains correct themes
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Children's social care" and text()="Children's social care"]
+
+    user checks page contains element    //*[@data-testid="Expand Details Section COVID-19" and text()="COVID-19"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Destination of pupils and students" and text()="Destination of pupils and students"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Early years" and text()="Early years"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Finance and funding" and text()="Finance and funding"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Further education" and text()="Further education"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Higher education" and text()="Higher education"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Pupils and schools" and text()="Pupils and schools"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section School and college outcomes and performance" and text()="School and college outcomes and performance"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Teachers and school workforce" and text()="Teachers and school workforce"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section UK education and training statistics" and text()="UK education and training statistics"]
+
+check publications have correct number of releases
+    # There are 56 releases on prod
```

Comment:
This should be `There are 56 publications that have a live release with a subject on prod`